.github/workflows/ci.yml

```diff
@@ -43,9 +43,31 @@ jobs:
       - name: Install package
         run: python -m pip install .[dev]
 
+      - name: Delete .git and .dvc folders
+        run: rm -rf .git .dvc
+
+      - name: Delete .dvc and .yaml files in data folder
+        run: |
+          find data -type f -name "*.dvc" -delete
+          find data -type f -name "*.yaml" -delete
+          find data -type f -name "*.yml" -delete
+
+      - name: Initialize Git
+        run: git init
+
+      - name: Initialize DVC
+        run: |
+          dvc init
+          dvc config core.autostage true
+
+      - name: Set up DVC local remote
+        run: |
+          mkdir dvc_local_remote
+          dvc remote add -d dvc_local_remote dvc_local_remote
+
       - name: Test package
         run: >-
-          python -m pytest -ra --cov --cov-report=xml --cov-report=term
+          python -m pytest -m "" -ra --cov --cov-report=xml --cov-report=term
```

Comment:
This is so that all tests, including those marked as "docker", are run on GH Actions. Not a problem to run those there since GH Actions runs in its own environment that is torn down at the end of the run (essentially the same as a container)